src/coreclr/vm/ceeload.cpp

```diff
@@ -2278,7 +2276,7 @@ ISymUnmanagedReader *Module::GetISymUnmanagedReader(void)
 
         // Note: we change the error mode here so we don't get any popups as the PDB symbol reader attempts to search the
         // hard disk for files.
-        lastErrorMode = SetErrorMode(SEM_NOOPENFILEERRORBOX|SEM_FAILCRITICALERRORS);
+        ErrorModeHolder errorMode{};
```

Comment:
Just for my education - what is `{}` good for? I think it would work just fine without it.

Reply:
This is part of [uniform initialization](https://isocpp.org/wiki/faq/cpp11-language#uniform-init). In this case, you're right it isn't needed and adds no more value. However, that is only because there is a user defined constructor. Using the `{}` for all forms means the type will be initialized in all cases - even if the user defined constructor is removed.

Reply:
Would you prefer I remove them?

Reply:
I do not have a strong opinion.